src/main/java/io/rubuy74/rhs/converter/EventDTOJSONConverter.java

```
@@ -0,0 +1,36 @@
+package io.rubuy74.rhs.converter;
+
+import io.rubuy74.rhs.dto.EventDTO;
+import io.rubuy74.rhs.utils.ValidatorUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Map;
+
+public class EventDTOJSONConverter {
+    public static Logger logger = LoggerFactory.getLogger(EventDTOJSONConverter.class);
+    public static EventDTO fromJson(Map<String,Object> rawPayload) {
+        if (rawPayload == null) {
+            logger.error("operation=deserialize_event_dto, " +
+                    "msg=Raw payload is null");
+            return null;
+        }
+
+        List<String> attributeList = List.of("event", "name", "date");
+        List<String> errorMessages = ValidatorUtils.checkAttributeList(rawPayload,attributeList);
+        if(!errorMessages.isEmpty()) {
+            errorMessages.forEach((errorMessage) -> {
+                logger.error("operation=deserialize_event_dto, msg={}", errorMessage);
+            });
+            return null;
```

maybe it's better to throw an exception for the failed parse